scripts/setup/dev_setup.sh

```diff
@@ -296,17 +296,6 @@ function install_cargo_binary {
 	fi
 }
 
-function install_toolchain {
-	version=$1
-	echo "==> Installing ${version} of rust toolchain..."
-	rustup install "$version"
```

Comment:
this step is needed for build-tool image

Reply:
Addressed by running `cargo version`

Reply:
For more background: https://note.xuanwo.io/#/page/Rustup

Reply:
this is needed for cross build, since it would only install x86 gnu

Reply:
Oh my mistake, cross toolchain setup is in separate command.

Reply:
so  `cargo version` is not needed.